website/docs/ref/enums.doc.js

```diff
@@ -0,0 +1,78 @@
+/* @flow */
+/*
+---
+id: enums
+title: Enums
+permalink: /docs/enums.html
+prev: react.html
```

Comment:
Talked about this with the team and we agreed on "Utility Types" as the name for all of these. Could you use that name here.